README/ReleaseNotes/v610/index.md

```diff
@@ -248,6 +248,7 @@ The following interfaces have been removed, after deprecation in v6.08.
 ## Parallelism and PROOF
 - Add ROOT::GetImplicitMTPoolSize function to get the size of the pool used to enable implicit multi threading
 - Add the TThreadExecutor::Foreach method for parallelising functions featuring void return type
+- Add TBufferMerger and TBufferMergerFile to facilitate writing in parallel to a single output file
```

Comment:
Sure, I just thought the release notes should be kept short due to the amount of things already there. How about adding the text from the commit message (shown below) here?

```
    Add TBufferMerger and TBufferMergerFile classes
    
    TBufferMerger is a class to facilitate writing data in
    parallel from multiple threads, while writing to a single
    output file. Its purpose is similar to TParallelMergingFile,
    but instead of using processes that connect to a network
    socket, TBufferMerger uses threads that each write to a
    TBufferMergerFile, which in turn push data into a queue
    managed by the TBufferMerger.
    
    These classes were originally written by Witold Pokorski and
    Philippe Canal for the GeantV project. There, they are named
    TThreadedMergingServer and TThreadedMergingFile, respectively.
    The imported code was then modified into the current version to
    be used initially by TDataFrame classes, and later by general
    ROOT users.
```